src/ActivitiesSDKAdapter.js

```diff
@@ -84,4 +92,35 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
 
     return this.activityObservables[ID];
   }
+
+  /**
+   * Creates an activity and returns an observable to the new activity data
+   *
+   * @param {object} activity  Adapter activity object
+   * @returns {Observable.<Activity>} Observable stream that emits data of the newly activity
+   */
+  postActivity(activity) {
+    const cardActivity = activity.card && [
+      {
+        contentType: 'application/vnd.microsoft.card.adaptive',
+        content: activity.card,
+      },
+    ];
+    const newActivity$ = from(this.datasource.messages.create({
+      roomId: activity.roomID,
+      text: activity.text,
+      attachments: cardActivity || activity.attachments,
+    })).pipe(
+      map(({id}) => ({
+        ID: id,
```

Comment:
it seems that this observable only emits the id of the newly posted activity, not a whole activity object, as described in the jsdoc. I think it's fine to just return the id (not an object containing the id).